src/reducers/CreateUserReducer.ts

```diff
@@ -33,29 +17,6 @@ export function create(
       return { ...state, createErrorMessage: action.data }
     case 'CLEAR_CREATE_ERROR_MESSAGE':
       return { ...state, createErrorMessage: null }
-    case 'CREATE_UPDATE_USERNAME':
-      return {
-        ...state,
-        username: action.data.username
-      }
-    case 'CREATE_UPDATE_PIN':
-      return {
-        ...state,
-        pin: action.data.pin,
-        pinErrorMessage: action.data.error
-      }
-    case 'AUTH_UPDATE_PASSWORD':
```

Comment:
Nobody dispatches this guy anymore. That means the `PasswordStatusState` will never update, which means the `<PasswordStatus>` component will never render anything interesting.

I see that you have moved the password status into local state on the `ChangePasswordScene`. This local state needs to be passed to the `<PasswordStatus>` component as a prop, so it will show something useful.